libs/expo/betterangels/src/lib/screens/Client/index.tsx

```diff
@@ -51,16 +51,26 @@ export default function Client({
   useLayoutEffect(() => {
     navigation.setOptions({
       headerLeft: () => (
-        <TextButton
-          regular
-          color={Colors.WHITE}
-          fontSize="md"
+        <Pressable
+          accessibilityRole="button"
+          accessible
           accessibilityHint="goes to previous screen"
-          title="Back"
           onPress={() =>
-            arrivedFrom ? router.navigate(arrivedFrom) : router.back()
+            arrivedFrom ? router.navigate(arrivedFrom) : router.navigate('/')
           }
-        />
+        >
+          <TextRegular color={Colors.WHITE}>Back</TextRegular>
+        </Pressable>
+      ),
+      headerRight: () => (
+        <Pressable
+          accessibilityRole="button"
+          accessible
+          accessibilityHint="goes to previous screen"
+          onPress={() => router.navigate(`/edit-client/${id}`)}
+        >
+          <TextRegular color={Colors.WHITE}>Edit</TextRegular>
+        </Pressable>
```

Comment:
<TextButton
          regular
          color={Colors.WHITE}
          fontSize="md"
          accessibilityHint="goes to Edit screen"
          title="Edit"
          onPress={() => router.navigate(`/edit-client/${id}`)}
        />